frontend/src/App.vue

```diff
@@ -327,12 +346,20 @@ export default {
         this.isReplaying = false;
       }
       const prompt = this.createPrompt(questions, name);
-      this.setPrompts([prompt]);
-
+	this.setPrompts([prompt]);
+	if (this.isGenerating) {
+		this.showButtons = true;
```

Comment:
true ?
shouldn't it be "this.showButtons = !this.isGenerating;"
or
if (!this.isGenerating) {

Reply:
When we already in writing state buttons should not be visible, but we need to show them again when there is a need to resolve conflict with generated files. The conflict (overwrite) steps appear in after we answered on all gen questions.
We can have several screens with conflicts and should be able to press on Back and Finish.